src/test/scala/mesosphere/mesos/ResourceMatcherTest.scala

```diff
@@ -1020,6 +1020,34 @@ class ResourceMatcherTest extends UnitTest with Inside with TableDrivenPropertyC
       resourceMatchResponse shouldBe a[ResourceMatchResponse.NoMatch]
     }
 
+    "not match offer with unknown disk type for persistent app" in {
+      val offer = MarathonTestHelper.makeBasicOffer()
+        .addResources(MarathonTestHelper.scalarResource("disk", 1024.0, disk = Some(MarathonTestHelper.rawDisk("/path"))))
+        .build()
+
+      val volume = VolumeWithMount(
+        PersistentVolume(
+          name = None,
+          persistent = PersistentVolumeInfo(
+            size = 128,
+            `type` = DiskType.Path)),
+        VolumeMount(None, "/var/lib/data"))
+
+      val app = AppDefinition(
+        id = "/test".toRootPath,
+        resources = Resources(
+          cpus = 1.0,
+          mem = 128.0,
+          disk = 0.0),
+        container = Some(Container.Mesos(
+          volumes = List(volume))),
+        versionInfo = OnlyVersion(Timestamp(2)))
+
+      val resourceMatchResponse = ResourceMatcher.matchResources(offer, app, knownInstances = Seq.empty, unreservedResourceSelector, config, Seq.empty, localRegion = None)
```

Comment:
@timcharper and @meln1k, this should fail with a match error since I commented that match for `RAW` and `BLOCK`. Yet, we get a match. Why is that?

Reply:
Nevermind. It was a typo on my side.